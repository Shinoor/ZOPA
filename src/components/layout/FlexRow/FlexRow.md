### Summary

`<FlexRow />` is meant to be a child of `<FlexContainer />`.

- If your goal is to make a nested layout, then it can also be a child of a `<FlexCol />` or any JSX element.
- **Be mindful** that it needs to have `<FlexCol />` as its direct child.
- See [`<FlexCol />`](/#/Components/Layout/FlexCol) docs for usage examples.
