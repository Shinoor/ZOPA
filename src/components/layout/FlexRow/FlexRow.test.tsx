import React from 'react';
import { render } from '@testing-library/react';
import FlexRow from './FlexRow';

describe('<FlexRow />', () => {
  it('renders a FlexRow component', () => {
    const { container } = render(<FlexRow />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
