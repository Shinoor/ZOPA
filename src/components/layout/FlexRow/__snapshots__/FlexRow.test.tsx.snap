// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlexRow /> renders a FlexRow component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  margin-right: -12px;
  margin-left: -12px;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

<div
  class="c0"
  cols="12"
  direction="row"
/>
`;
